### PR TITLE
Doxgyen workaround for getting namespace methods to show up

### PR DESCRIPTION
**Pull request by jonwong12**
Certain namespace methods such as `serac::buildMeshFromFile` don't show up in the doxygen search. Apparently the namespace itself needs to be "documented" for doxygen to show the doxygen comments for namespace methods. [stackoverflow](https://stackoverflow.com/questions/25347174/doxygen-no-detailed-description-for-functions-in-namespaces)

### CODE REVIEW

**Review by white238 — APPROVED** [review]

File: src/serac/numerics/mesh_utils.hpp [hunk 1]
```diff
@@ -19,6 +19,9 @@
 #include "mfem.hpp"
 #include "serac/infrastructure/input.hpp"
 
+/**
+   The Serac namespace
```
Comment (joshessman-llnl) [inline src/serac/numerics/mesh_utils.hpp]:
```suggestion
 * The Serac namespace
```

Comment (joshessman-llnl) [reply src/serac/numerics/mesh_utils.hpp]: For consistency with the rest of our documentation

**Review by joshessman-llnl — APPROVED** [review]

**Comment by white238** [comment]
https://serac.readthedocs.io/en/feature-jonwong12-doxygen_workaround/

turned on your branch so you can see it

Edit: of course now I'm not sure if we have a link to Doxygen docs there and/or possibly build them.  This is too much thought for my vacation day.

**Comment by jamiebramwell** [comment]
We do autobuild the doxygen with the sphinx on read the docs. The appropriate version is available [here](https://serac.readthedocs.io/en/feature-jonwong12-doxygen_workaround/doxygen/html/index.html).

**Review by jamiebramwell — APPROVED** [review]
Thanks for figuring this out @jonwong12 ! Let me know when you think it's ready to merge.

**Comment by jamiebramwell** [comment]
@jonwong12 do you think this is ready to merge?

**Comment by jonwong12** [comment]
@jamiebramwell Yes.

**Comment by jamiebramwell** [comment]
LGTM

**Pull request merged**